posydon/utils/x_ray_luminosities.py

```diff
@@ -0,0 +1,188 @@
+
+"""Determins XRB type and calculate X-ray luminosities."""
+
+
+__authors__ = [
+    "Devina Misra <devina.misra@unige.ch>",
+]
+
+
+import os
+import numpy as np
+from scipy.interpolate import interp1d
+from scipy.optimize import newton
+from scipy.integrate import quad
+from posydon.utils import constants as const
+import copy
+import warnings
+from scipy.interpolate import PchipInterpolator
+from posydon.utils.common_functions import eddington_limit
+
+
+PATH_TO_POSYDON = os.environ.get("PATH_TO_POSYDON")
+
+
+# def xrb_type(binary, idx=-1):
+#     if binary.state in ['detached', 'RLO1', 'RLO2']:
+#         if binary.star_1.state in ['NS', 'BH']:
+#             accretor = binary.star_1
+#             donor = binary.star_2
+#             don_rel_RL = binary.rl_relative_overflow_2
+#         elif binary.star_2.state in ['NS', 'BH']:
+#             accretor = binary.star_2
+#             donor = binary.star_1
+#             don_rel_RL = binary.rl_relative_overflow_1
+#         else:
+#             xrb_type = 0
+#             return xrb_type
+#     else:
+#         xrb_type = 0
+#         return xrb_type
+
+#     if don_rel_RL is None:
+#         xrb_type = 0
+#         return xrb_type
+
+#     don_RL = 10**donor.log_R / (don_rel_RL + 1)
+#     if ( (donor.state == 'H-rich_Core_H_burning') & (donor.mass >= 3.0) &
+#         (binary.state == 'detached') & (don_RL <= (100.0 * 10 ** donor.log_R) ) & 
+#         (donor.surf_avg_omega_div_omega_crit >= 0.7) & (binary.orbital_period >= 10.0) & (binary.orbital_period <= 300.0) ) :
+#             xrb_type = 3
+#     elif ((binary.state =='RLO1') & (binary.state =='RLO2')):
+#             xrb_type = 1
+#             print(binary.state , donor.state, accretor.state)
+#     else:
+#             xrb_type = 2
+
+#     return xrb_type
+
+
+def x_ray_luminosity(binary, observe_wind_XRB=False, idx=-1):
+    """ Calculate the geometrical beaming of a super-Eddington accreting source.
+    Compute the super-Eddington isotropic-equivalent accretion rate and the
```

Comment:
Good doc strings! Just add a white line here between the "Calculate ..." line and "Compute ...." line

Could you add a description of the other input parameters too? ie:`observe_wind_XRB` and `idx`.